docs-beta/src/components/Features.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import "../css/tailwind-styles.css"; 
```

Comment:
I think you only need to include this one just once... (this is repeated in many components) - the import from the pages/index.js should be good enough!

Reply:
I needed to import React for all of them but not the styles -- just fixed that!